src/Microsoft.AspNetCore.WebHooks.Receivers/Filters/WebHookVerifyBodyTypeFilter.cs

```diff
@@ -125,6 +132,70 @@ public void OnResourceExecuted(ResourceExecutedContext context)
             // No-op
         }
 
+        /// <summary>
+        /// Determines whether the specified request contains JSON as indicated by a content type of
+        /// <c>application/json</c>, <c>text/json</c> or <c>application/xyz+json</c>. The term <c>xyz</c> can for
+        /// example be <c>hal</c> or some other JSON-derived media type.
+        /// </summary>
+        /// <param name="request">The <see cref="HttpRequest"/> to check.</param>
+        /// <returns>
+        /// <see langword="true"/> if the specified request contains JSON content; otherwise, <see langword="false"/>.
+        /// </returns>
+        protected static bool IsJson(HttpRequest request)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            var contentType = request.GetTypedHeaders()?.ContentType;
```

Comment:
When would this ever return null?

Reply:
I probably checked for `GetTypedHeaders()` returning `null` based on the older Web API code this morphed from. I'll remove the question mark.

Or, were you asking about the next line's `null` check of `contentType`?